modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -103,135 +66,132 @@ The repositories and channels for synchronizing upstream content must be configu
+. Click btn:[Create Repository] and set these parameters for the ``main`` repository:
+* In the [guimenu]``Repository Label`` field, type [systemitem]``ubuntu-bionic-main``.
+* In the [guimenu]``Repository URL`` field, type [systemitem]``http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/``.
+* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+* Leave all other fields as the default values.
+. Click btn:[Create Repository]
+. Click btn:[Create Repository] and set these parameters for the ``main-updates`` repository:
+* In the [guimenu]``Repository Label`` field, type [systemitem]``ubuntu-bionic-main-updates``.
+* In the [guimenu]``Repository URL`` field, type [systemitem]``http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/``.
+* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+* Leave all other fields as the default values.
+. Click btn:[Create Repository].
 
-* Repository Label: ubuntu-bionic-main-updates
-* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/
-* Repository Type: deb
 
-. Create custom channels under the `pool` channel, mirroring the vendor channels.
+
+When you have created the repositories, you can create the custom channels.
+
 Ensure the custom channels you create have `AMD64 Debian` architecture.
-Create this structure:
-+
-----
- ubuntu-18.04-pool for amd64 (vendor channel)
- |
- +- Ubuntu-18.04-SUSE-Manager-Tools for amd64 (vendor channel)
- |
- +- ubuntu-18.04-amd64-main (custom channel)
- |
- +- ubuntu-18.04-amd64-main-updates (custom channel)
-----
 
-. Associate the custom channels with the appropriate custom repositories.
-. Synchronize the new custom channels.
-You can check the progress of your synchronization from the command line with this command:
-+
-----
-tail -f /var/log/rhn/reposync.log /var/log/rhn/reposync/*
-----
-. To use bootstrap with {ubuntu}, you will need to create a bootstrap repository.
-You can do this from the command line with [command]``mgr-create-bootstrap-repo``:
-+
-----
-mgr-create-bootstrap-repo --with-custom-channels
-----
+Your custom channels should use this structure:
+
+* Parent channel: [systemitem]``ubuntu-18.04-pool for amd64``
+* Child vendor channel: [systemitem]``Ubuntu-18.04-SUSE-Manager-Tools for amd64``
+* Child custom channel: [systemitem]``ubuntu-18.04-amd64-main``
+* Child custom channel: [systemitem]``ubuntu-18.04-amd64-main-updates``
 
-For more information on creating custom repositories, see xref:administration:channel-management.adoc[].
+When you have the channels set up, associate each channel with the appropriate repository, and synchronize then channels.
+
+[IMPORTANT]
+====
+You need all the new channels fully syncronized before bootstraping any Ubuntu client.
+====
 endif::[]
 
 
 
-.Procedure: Preparing to Register {ubuntu} Clients with Spacewalk
+ifeval::[{uyuni-content} == true]
```

Comment:
I think this ifeval is problematic.

Reply:
I think you mean:
```
== Repository and Channel Management
```
Is not really needed. Right?

Please try to be more specific.

As for the other minor issues, I accepted the change but -as discussed- please fix such things on your own.

Reply:
Or even that title show go above.

Reply:
Should be fixed now.

Reply:
See the PDFs below.